src/Components/Onboarding/index.tsx

```diff
@@ -1,10 +0,0 @@
-import React from "react"
-
-import Wizard from "./Wizard"
-
-import Artists from "./Steps/Artists"
-import CollectorIntent from "./Steps/CollectorIntent"
-
-export default () => {
-  return <Wizard stepComponents={[CollectorIntent, Artists]} />
-}
```

Comment:
Didn't seem like this file was being used anywhere -- let me know if you'd like me to add it back for some reason.